### DISCUSSION TITLE
Network Graph not loading

### DISCUSSION THREAD

**Discussion by MelissaMolinari** [community/community#18526] [Other Feature Feedback, Questions, & Ideas]
I was working normally, made a merge (git merge --no-ff orgin branchname) betwen two branches and when i went lo check the graph it wasnt loading anymore.

Is there anything i can do to fix this? o is there a diferent way to viasualize the graph? Our group is fairly new to working with git (were students)

![image](https://user-images.githubusercontent.com/89586902/173448361-d0cc975d-b816-4889-a534-1168e4f2a3a4.png)

**Comment by MathiasReker** [comment]
Try another browser.

**Comment by 5HT2** [comment]
Try another browser, or force refresh (Ctrl Shift R) the page.

**Comment by matt-cos** [comment]
I often have to hard refresh a few times to get the graph to show up. I also find that I am more likely to get stuck with the spinner if I have recently made changes to the remote.

**Comment by jfullstackdev** [comment]
I think that's a known issue, as revealed by this search

https://github.com/orgs/community/discussions?discussions_q=network+graph

but I just tested that by now, I'm on Chrome and there is no lag displaying the data

**Comment by Solution-Builders-Dev9** [comment]
It's happening to our repo after merging pull request.
![image](https://user-images.githubusercontent.com/81647389/231869251-a0a8d299-217a-4bbc-aee3-4ff13021db47.png)
